static/swagger/eds-spec.json

```diff
@@ -0,0 +1,370 @@
+      "description": "APIs related to event subscriptions and streaming."
+    },
+    {
+      "name": "EDS Metrics",
+      "description": "APIs for retrieving metrics data related to EDS."
+    },
+    {
+      "name": "EDS Status",
+      "description": "APIs for checking the health and version of EDS."
+    }
+  ],
+  "paths": {
+    "/fabric/{fabricName}/api/es/v1/stream/{streamName}/publish": {
+      "post": {
+        "summary": "Asynchronous Publish",
+        "description": "Publish events to a specified stream asynchronously.",
+        "tags": [
+          "Event Publish"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "fabricName",
+            "required": true,
+            "schema": {
+              "type": "string"
+            },
+            "description": "The name of the fabric."
+          },
+          {
+            "in": "path",
+            "name": "streamName",
+            "required": true,
+            "schema": {
+              "type": "string"
+            },
+            "description": "The name of the stream."
+          },
+          {
+            "in": "query",
+            "name": "type",
+            "required": true,
+            "schema": {
+              "type": "string"
+            },
+            "description": "The type of the stream (local or global or collection)."
+          }
+        ],
+        "requestBody": {
+          "required": true,
+          "description": "The request body containing the event data to publish to the stream.",
+          "content": {
+            "application/json": {
+              "schema": {}
+            }
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "Successful response.",
+            "headers": {
+              "x-gdn-service": {
+                "description": "The hostname of the GDN.",
+                "schema": {
+                  "type": "string",
+                  "description": "Custom header value indicating the hostname of the GDN."
+                }
+              },
+              "X-PhotonIQ-ES": {
+                "description": "Epoch time header.",
+                "schema": {
+                  "type": "string",
+                  "description": "Epoch time in seconds."
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad request.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "error": {
+                      "type": "string",
+                      "description": "Error message."
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "ws://api/es/v1/subscribe": {
+      "get": {
+        "summary": "Subscribe to Stream via Web Socket",
```

Comment:
```suggestion
        "summary": "Subscribe to stream via Web Socket",
```

Reply:
Maybe I alread menion this, but maybe drop the via web socket.  This is a summary.  If you must bring up web socket, then put it in the body of description

Reply:
Done